app/code/Magento/Catalog/Model/Product/Image/Process/Queue.php

```diff
@@ -0,0 +1,323 @@
+<?php
```

Comment:
Please review method visibility in this class, it looks like a lot of them need to be `private`.